images/access-setup/content/bin/setup_compute.sh

```diff
@@ -176,6 +178,28 @@ tekton_chains_manifest(){
   printf "OK\n"
 }
 
+tekton_results_manifest(){
+  manifest="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
+  if [ ! -e "$manifest" ]; then
+    manifests_tmp_dir="$(dirname "$manifest")/tmp"
+    mkdir -p "$manifests_tmp_dir"
+    read -p "Please enter database username: " database_user
+    read -p "Please enter database password: " database_password
```

Comment:
Add `-s` for secret. You probably need `-r` as well so that shellcheck does not complain.

Reply:
added both options